data_ingestion/common/census.py

```diff
@@ -1,7 +1,7 @@
 from .di_url_file_to_gcs import url_file_to_gcs
```

Comment:
Could this file name maybe be census_ingestion_helper or census_util or something?

Also would we want these data ingestion helper/util files to be by data source, data set or by final big query table? I can see a case for each. Not familiar with the structure of census data but I think if the state names and household income were in different BQ tables it may make sense to keep their helper stuff separate

Reply:
My 2 cents is that since the data_ingestion cloud function is only responsible for getting data from different sources and putting them in buckets, we should split those by data source. For the function that uploads to BQ, I would split that out by BQ table. The reason I put the census stuff together is there are some common aspects to the census API (similar query params, similar data structure, etc). Having those in one file allows sharing code more easily, and it's always possible to split it out later if we see the need.

I really know about python/cloud naming conventions are. I'm used to working in Google Java and JS, where you pretty much always make classes for your file-level exports and you name the file the same as the class. I guess I would lean towards census_ingestion?

Reply:
Ingestion is already in the path name so I wouldn't want to repeat it...
Mine is going to be called "County Health Rankings"
Maybe it's fine if it's just the name of the data source. We can do whatever for now since it's a prototype and if at the end we wish we'd done it different, we can do it differently later, right!

Reply:
Haha okay, I'll change back to census. If we end up splitting out the different census parts we can rename them.